### PR TITLE
Make Series.map() documentation a bit more verbose

### PR DESCRIPTION
**Pull request by manuels**

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -2138,6 +2138,14 @@ def map(self, arg, na_action=None):
         two   bar
         three baz
 
+        >>> z
```
Comment (jorisvandenbossche) [inline pandas/core/series.py]: Can you add a single sentence between the examples? Like "Mapping a dictionary works similar as with a Series (mapping dict keys on index labels): "

Comment (jorisvandenbossche) [inline pandas/core/series.py]: Could you add a single sentence between the examples? Eg something like "Mapping a dict works similarly to mapping a Series (the dict keys are mapped on the index labels): "

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks a lot!

Now you are at it, could you also add the construction of the objects used in the examples? (so the example can be copy pasted to try out yourself)

**Comment by manuels** [comment]
Thanks for the quick response. I have added some text and added more code to be able to reproduce 'at home'.

**Comment by jreback** [comment]
thanks!

**Pull request closed without merging**